test/gtest/ucs/test_mpool.cc

```diff
@@ -73,6 +83,39 @@ class test_mpool : public ucs::test {
         return UCS_LOG_FUNC_RC_CONTINUE;
     }
 
+    static bool
```

Comment:
return void; use EXPECT/ASSERT macros in the body of this function instead of returning true/false

Reply:
changed accordingly.